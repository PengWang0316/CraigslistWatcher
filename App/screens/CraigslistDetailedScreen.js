import React from 'react';
import { SafeAreaView } from 'react-navigation';

import DarkGreenStatusBar from '../components/DarkGreenStatusBar';
import ImageCarousel from '../components/ImageCarousel';
import Styles from './Styles';

const CraigslistDetailedScreen = props => (
  <SafeAreaView style={Styles.mainView}>
    <DarkGreenStatusBar />
    <ImageCarousel dataId={props.navigation.state.dataId} />
  </SafeAreaView>
);
export default CraigslistDetailedScreen;
