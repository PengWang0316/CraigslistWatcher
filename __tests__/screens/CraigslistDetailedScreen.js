import React from 'react';
import renderer from 'react-test-renderer';

import CraigslistDetailedScreen from '../../App/screens/CraigslistDetailedScreen';

jest.mock('../../App/components/DarkGreenStatusBar', () => 'DarkGreenStatusBar');
jest.mock('../../App/components/CraigslistDetailPage', () => 'CraigslistDetailPage');

describe('Test CraigslistDetailedScreen', () => {
  const defaultProps = {
    state: {
      params: {
        dataId: 'dataId',
        url: 'url'
      }
    }
  };
  test('CraigslistDetailedScreen Snapshot', () => expect(renderer.create(<CraigslistDetailedScreen navigation={defaultProps} />).toJSON()).toMatchSnapshot());
});
