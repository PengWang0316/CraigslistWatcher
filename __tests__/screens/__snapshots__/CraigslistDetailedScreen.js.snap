// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test CraigslistDetailedScreen CraigslistDetailedScreen Snapshot 1`] = `
<View
  collapsable={undefined}
  onLayout={[Function]}
  pointerEvents="box-none"
  style={
    Object {
      "backgroundColor": "#e0e0e0",
      "flex": 1,
      "paddingBottom": 0,
      "paddingLeft": 0,
      "paddingRight": 0,
      "paddingTop": 20,
    }
  }
>
  <DarkGreenStatusBar />
  <CraigslistDetailPage
    dataId="dataId"
    url="url"
  />
</View>
`;
